src/perform_step/low_storage_rk_perform_step.jl

```diff
@@ -76,14 +77,15 @@ end
   integrator.destats.nf += 1
 end
 
-mutable struct WilliamsonWrapper{kType, dtType}
+mutable struct WilliamsonWrapper{kType, dtType, coffType}
   kref::kType
   dt::dtType
+  coefficient::coffType
 end
 
-@inline Base.setindex!(a::WilliamsonWrapper{kType, dtType}, b::bType, c::cType) where {kType, dtType, bType, cType} = (a.kref[c] += a.dt * b)
+@inline Base.setindex!(a::WilliamsonWrapper{kType, dtType}, b::bType, c::cType) where {kType, dtType, bType, cType} = (a.kref[c] = a.coefficient * a.kref[c] + a.dt * b)
```

Comment:
What's the purpose of all these type annotations and where clauses here? It seems only `WilliamsonWrapper` is needed?